torchvision/prototype/transforms/functional/_type_conversion.py

```diff
@@ -22,4 +22,4 @@ def decode_video_with_av(encoded_video: torch.Tensor) -> Tuple[torch.Tensor, tor
 
 
 def label_to_one_hot(label: torch.Tensor, *, num_categories: int) -> torch.Tensor:
-    return cast(torch.Tensor, one_hot(label, num_classes=num_categories))
+    return one_hot(label, num_classes=num_categories)  # type: ignore[no-any-return]
```

Comment:
Question: Is mypy complaining because it can't tell `LongTensor` is `Tensor`?

Reply:
`mypy` infers `Any` as return type of `one_hot`.

Reply:
Yeah I can see why mypy can't read the type. [Here](https://github.com/pytorch/pytorch/blob/28339ddc252325a1fc93e3302e9c4f93cdc0034b/torch/nn/functional.py#L4477-L4480) is the code on PyTorch.